File: Service/Implement/DanhMucChiTieuMoiTruongService.cs
namespace Service.Implement
{
    public class DanhMucChiTieuMoiTruongService : BaseService<DanhMucChiTieuMoiTruong, IDanhMucChiTieuMoiTruongRepository>
    , IDanhMucChiTieuMoiTruongService
    {
        private readonly IDanhMucChiTieuMoiTruongRepository _DanhMucChiTieuMoiTruongRepository;
        public DanhMucChiTieuMoiTruongService(IDanhMucChiTieuMoiTruongRepository DanhMucChiTieuMoiTruongRepository) : base(DanhMucChiTieuMoiTruongRepository)
        {
            _DanhMucChiTieuMoiTruongRepository = DanhMucChiTieuMoiTruongRepository;
        }
    }
}

File: CMS/src/app/shared/DanhMucChiTieuMoiTruong.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucChiTieuMoiTruong } from 'src/app/shared/DanhMucChiTieuMoiTruong.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucChiTieuMoiTruongService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucChiTieuMoiTruong";
    }
}

